modules/packages/Python.chpl

```diff
@@ -1930,6 +1942,20 @@ module Python {
       return res;
     }
 
+    /*
+      Returns the type of the object.
+
+      Equivalent to calling ``type(obj)`` in Python.
+    */
+    proc pyType(): owned Value throws {
```

Comment:
I would note the addition of this and retain in the PR message

Reply:
Oooh yeah I forgot to add that for `pyType`. I'm not super happy with the name, but since `type` is reserved I don't have a better one

retain is intentionally nodoc and arguably just an implementation detail for the operators